opm/autodiff/ISTLSolverEbos.hpp

```diff
@@ -381,10 +528,48 @@ namespace Opm
             }
         }
     protected:
+
+        bool isParallel() const {
+            return parallelInformation_.type() == typeid(ParallelISTLInformation);
```

Comment:
Needs #if HAVE_MPI for the ParallelISTLInformation class to exist, #else branch can return false.

Reply:
Thanks for the catch